### ISSUE TITLE
Validator rules behave differently depending on type

### ISSUE THREAD

**Issue by luketarplin** [the-support-group/request-validator#6]
Rules such as max & min behave differently depending on the type of data being checked, this is a potential to allow incorrect values through the validator as successful if the usage neglects the type. eg: max:5 with data 6 would be valid whereas the intention was to set a max value of 5!

**Comment by forceedge01** [comment]
@luketarplin Good catch, this issue will probably result in creating two separate rules for integers and strings. Feel free to create a pull request with your version of the solution if possible. 👍
